docker/Dockerfile

```diff
@@ -4,7 +4,11 @@ ARG TARGETPLATFORM
 ARG BUILDPLATFORM
 
 RUN echo "I am running on $BUILDPLATFORM, building for $TARGETPLATFORM"
-RUN mkdir -p /etc/fireguard /etc/wireguard
+RUN mkdir -p /etc/fireguard /etc/wireguard && \
+    apt-get update && \
+    apt-get install -y git && \
+    apt-get -y clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I'll merge this and we can try, but I believe this will break aarch64 build inside docker buildx, let's see :)